libpodpy/__init__.py

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Can you stick all the libpodpy stuff in `contrib/` to distinguish it from the main repo?